how-to/donate_project.md

```diff
@@ -0,0 +1,111 @@
+# Project Donation Application
+
+Please see the [proposals/new-project-donation-process.md](Documentation) to
+better understand the full process for submitting a new project.
+In short, copy this Application Template and populate the document.
+--!>
+
+## CNCF checklist
+- [ ] All project metadata and resources are vendor-neutral
+- [ ] Governance structure
+- [ ] Contributing guides
+- [ ] Overlap with existing Kubeflow projects
+- [ ] Public list of adopters.
+
+
+## Background information
+
+1. Submitter Name
+    -
+
+1. Submitter’s relationship to project / title
+    -
+
+1. Project Name
+    -
+
+1. Why this project is valuable to the Kubeflow Community
+    -
+
+1. List of integrations with Kubeflow Core components (and potential)
+    -
+
+1. Short Description / Functionality
+    -
+
+1. Website
+    -
+
+1. GitHub repository
+    -
+
+1. 1st Release date
+    -
+
+1. Project Meeting Times
+    -
+
+1. Meeting Notes
+    -
+
+1. Installation Documentation
+    -
+
+1. Project Documentation
+    -
+
+1. Security Profile (CVE scanning)
+    -
+
+1. Ownership / Legal Profile (license type, any potential issues for CNCF)
+    -
+
+1. Authorization, Isolation mechanisms
+    -
+
+1. Project Roadmap
+    -
+
+1. Other Information
+    -
+
+## Metrics
+
+- Number of Maintainers
+- Number of Releases in last 12 months
+- Number of Contributors
+- Number of Users
+- Number of Forks
+- Number of Stars
+- Number of package/project installations/downloads
+
+## Kubeflow Specific Questions
+
+1. Manifest Integration
+    - [ ] Yes
+    - [ ] No
+    - [ ] Planned
+
+1. Installation
+    - [ ] Testing
+    - [ ] Stand alone
+    - [ ] Part of Manifests
+    - [ ] Part of Distributions
+
+1. Installation Documentation (Current Quality)
+    - [ ] Good
+    - [ ] Fair
+    - [ ] Part of Kubeflow
```

Comment:
This is not applicable, since we remove all docs for non-Kubeflow projects from the website.

Reply:
This would be from the lens of the component itself.

Reply:
i.e., how do I install this project

Reply:
Do you mean where projects want to maintain their documentations if Kubeflow community accept the project?

Reply:
I think it's more of "How is the quality of the current documentation?"